url.bs

```diff
@@ -85,6 +85,170 @@ valid input. User agents, especially conformance checkers, are encouraged to rep
  unclear to other developers.
 </div>
 
+<table>
+ <thead>
+  <tr>
+   <th>Error Code
+   <th>Error Message
+ <tbody>
+  <tr>
+   <td><dfn>unexpected-c0-control-or-space</dfn>
+   <td>This error occurs when the input to the URL parser contains leading or trailing
+   <a>C0 control or space</a> code points. The URL parser subsequently strips any matching code
+   points.
+  <tr>
+   <td><dfn>unexpected-ascii-tab-or-newline</dfn>
```

Comment:
I would prefer uppercasing ASCII here. Or is there a reason for all lowercase names? Similar for IPv4 and such.

Reply:
No particular reason. Was mostly following what was done for the HTML [parse errors](https://html.spec.whatwg.org/multipage/parsing.html#parse-errors) since they were all lowercase and didn't use uppercase for terms like "HTML" or "EOF".

Reply:
Thanks, @domenic what do you think? Should we adjust HTML a bit there?

Reply:
I like the all-lowercase style personally.

In general really unclear what these identifiers are to be used for and what conventions they should follow, so we're just making up our own rules. Other systems might use numeric error codes, or SHOUTY_CASE variants which could be used as variable names in most programming languages. Or, we could ask, why are we using dashes instead of spaces?

Similar questions have come up in the unrelated context of Web IDL enumerations. There we've mostly settled on lower-kebab-case, I think.

Reply:
I see, but these are non-string enumerations. I guess I can go along with that.

Reply:
Is the consensus to stick with all lowercase?

Reply:
Yeah.